examples/rmq_broadcast_client.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+import sys
+
+from kiwipy import rmq
+
+# pylint: disable=invalid-name
+
+body = ' '.join(sys.argv[1:]) or "___"
+
+with rmq.RmqThreadCommunicator.connect(connection_params={'url': 'amqp://127.0.0.1'}) as communicator:
+    # send message with different sender and subject
+
+    # listen by two subscriber
+    sendr = 'bob.jones'
+    subj = 'purchase.car'
+    communicator.broadcast_send(body, sender=sendr, subject=subj)
```

Comment:
maybe even simple/shorter to pass the strings directly here since you are anyhow not reusing the variables

Reply:
I think using variables here make it more clear to readers.

Reply:
ok, as you prefer